Dockerfile

```
@@ -27,4 +27,4 @@ RUN stack setup \
             -- --version \
     # Jenkins runs the builds not as root, but with a system dependent user id.
     # Therefore, we allow all users to access the stack cache.
-    && chmod -R a+rw $STACK_ROOT
+    && chmod -R a+rw $STACK_ROOT
```

```suggestion
    && chmod -R a+rw $STACK_ROOT

```
Missing newline at end of file (I think)